spec/facility_spec.rb

```diff
@@ -23,4 +31,79 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe 'Iteration 2, Vehicle Registration' do
+    it '#register_vehicle' do
+      expect(@facility_1.registered_vehicles).to eq([])
+
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility_1.collected_fees).to eq(100)
+    end
+
+    it 'register multiple vehicles' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@camaro)
+      @facility_1.register_vehicle(@bolt)
+      
```

Comment:
To make this test more robust we could also test that @facillity_1.registered_vehicles now holds all these vehicles in that array.